core/creators/themedui.js

```diff
@@ -239,6 +239,9 @@ CKEDITOR.replaceClass = 'ckeditor';
 
 			// Delay to avoid race conditions (setMode inside setMode).
 			setTimeout( function() {
+				if ( editor.status === 'destroyed' || editor.container.isDetached() ) {
```

Comment:
Actually this case is supposed to be covered by
`test editor set mode when editor is detached'`
and
`test editor set mode when editor is destroyed'`

But these tests are false positive, because they use fictional mode :sadfrog:.